workos/user_management.py

```diff
@@ -476,6 +481,9 @@ def get_authorization_url(
             params["login_hint"] = login_hint
         if state is not None:
             params["state"] = state
+        if code_challenge is not None:
```

Comment:
is an empty string valid? if not you can switch this to `if code_challenge:`

Reply:
Updated ✅